tests/test_pipreqs.py

```diff
@@ -9,18 +9,28 @@
 """
 
 from io import StringIO
-from unittest.mock import patch
+import logging
+from unittest.mock import patch, Mock
 import unittest
 import os
 import requests
 import sys
+import warnings
 
 from pipreqs import pipreqs
 
 
 class TestPipreqs(unittest.TestCase):
-    def setUp(self):
-        self.modules = [
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
why rename self to cls? what cls means? class? not sure why this is necessary or what benefits it brings